packages/tempus/src/Tempus_SolutionState_decl.hpp

```diff
@@ -125,18 +125,23 @@ class SolutionState :
       { TEUCHOS_ASSERT(metaData_nc_ != Teuchos::null);
         return metaData_nc_; }
 
-    virtual Scalar getTime()     const {return metaData_->getTime();}
-    virtual int getIndex()    const {return metaData_->getIStep();}
-    virtual Scalar getTimeStep() const {return metaData_->getDt();}
-    virtual Scalar getOrder()    const {return metaData_->getOrder();}
-    virtual Scalar getNRunningFailures() const
-      { return metaData_->getNRunningFailures(); }
-    virtual Scalar getNConsecutiveFailures() const
-      { return metaData_->getNConsecutiveFailures(); }
-    virtual Status getSolutionStatus() const
-      { return metaData_->getSolutionStatus(); }
-    virtual bool getOutput()     const {return metaData_->getOutput();}
-    virtual bool getIsSynced()   const {return metaData_->getIsSynced();}
+    virtual Scalar getTime()             const {return metaData_->getTime();}
+    virtual int    getIndex()            const {return metaData_->getIStep();}
+    virtual Scalar getTimeStep()         const {return metaData_->getDt();}
+    virtual Scalar getErrorAbs()         const {return metaData_->getErrorAbs();}
+    virtual Scalar getErrorRel()         const {return metaData_->getErrorRel();}
+    virtual Scalar getOrder()            const {return metaData_->getOrder();}
+    virtual Scalar getNFailures()        const {return metaData_->getNFailures();}
+    virtual Scalar getNRunningFailures() const {return metaData_->getNRunningFailures();}
+    virtual Scalar getNConsecutiveFailures() const {return metaData_->getNConsecutiveFailures();}
```

Comment:
These should be `int` type

Reply:
Ahh crude.  copy-paste errors.  :(  Thanks for catching this!